js/views/playingScreen.js

```diff
@@ -187,9 +188,11 @@ var PlayingScreen = React.createClass({
     if (!this.isMobile) {
       e.stopPropagation(); // W3C
       e.cancelBubble = true; // IE
-      if (!this.props.controller.videoVr && !this.props.controller.state.accessibilityControlsEnabled) {
-        this.props.controller.togglePlayPause();
+      var isSelectableScreen = Utils.areArgumentsEqual(e.target.className, CONSTANTS.CLASS_NAMES.SELECTABLE_SCREEN);
+      if (!this.props.controller.videoVr && isSelectableScreen) {
+        this.props.controller.togglePlayPause();//if clicked on selectableSceen
       }
+      //the order of the loop and this.props.controller.state is not important
       this.props.controller.state.accessibilityControlsEnabled = true;
```

Comment:
Can we take this line out? Because it's turning accessbility on regardless of it was enabled before this.